src/lib.rs

```diff
@@ -51,15 +51,16 @@ pub use crate::error::Error;
 
 use serde_json::value::RawValue;
 
-/// Shorthand method to convert an argument into a [Box<serde_json::value::RawValue>].
-/// Since serializers rarely fail, it's probably easier to use [arg] instead.
+/// Shorthand method to convert an argument into a [`Box<serde_json::value::RawValue>`].
```

Comment:
Does this work? I thought links can only point to identifiers. Taking user to `Box` is not very useful here, I think.

Reply:
Woops, changed to "... a boxed [serde_json::value::RawValue]" (but with backticks)